### PR TITLE
Rails 5

### PR DESCRIPTION
**Pull request by victorperez**
This pull request add Rails 5 Compatibility.

### CODE REVIEW

File: lib/themes_on_rails/action_controller.rb [hunk 1]
```diff
@@ -44,6 +44,8 @@ def before_filter_method(options)
             options.delete(:prepend) ? :prepend_before_filter : :before_filter
           when 4
             options.delete(:prepend) ? :prepend_before_action : :before_action
+          when 5
```
Comment (dfreerksen) [inline lib/themes_on_rails/action_controller.rb]: You can probably just to `when 4, 5` instead of having a separate `when` for Rails 5

**Comment by victorperez** [comment]
@chamnap any chance that this gets merged some day?

**Comment by darkcode85** [comment]
@chamnap could you bump the version? Thanks

**Comment by victorperez** [comment]
@chamnap Thanks to merge it! Yes please could we bump the version?

**Comment by chamnap** [comment]
Just bump the version. Thanks.

**Comment by victorperez** [comment]
@chamnap Thanks! :+1:

**Pull request merged**